projects/igniteui-angular/src/lib/directives/template-outlet/template_outlet.directive.ts

```diff
@@ -33,7 +33,7 @@ export class IgxTemplateOutletDirective implements OnChanges {
      * The embedded views cache. Collection is key-value paired.
      * Key is the template id, value is the embedded view for the related template.
      */
-    private _embeddedViewsMap: Map<string, EmbeddedViewRef<any>> = new Map();
+    private _embeddedViewsMap: Map<string, Map<any, EmbeddedViewRef<any>>> = new Map();
```

Comment:
You can also update the description above the collection so that it's up to date.